code/packaging/app-config/src/main/resources/META-INF/cattle/iaas-api/spring-iaas-api-context.xml

```diff
@@ -519,8 +519,27 @@
 	
     <bean id="TokenAccountLookup" class="io.cattle.platform.iaas.api.auth.integration.internal.rancher.TokenAccountLookup" />
 
+    <bean id="NoAuthenticationProxy" class="io.cattle.platform.iaas.api.request.handler.GenericWhitelistedProxy" >
+        <property name="noAuthProxy">
+               <value type="java.lang.Boolean">true</value>
+        </property>
+        <property name="allowedPaths">
+           <list>
+               <value>/v1-auth/saml/</value>
+           </list>
+        </property>
+    </bean>
 
-    <bean class="io.cattle.platform.iaas.api.request.handler.GenericWhitelistedProxy" />
+    <bean id="AuthenticatedProxy" class="io.cattle.platform.iaas.api.request.handler.GenericWhitelistedProxy" >
+        <property name="noAuthProxy">
```

Comment:
I will test again, but I got an error loading bean for missing property noAuthProxy